mos-platform/atari8-stdcart/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+platform(atari8-stdcart COMPLETE PARENT atari8)
+
+if(NOT CMAKE_CROSSCOMPILING)
+  return()
+endif()
+
+install(FILES
+  cart-rom-fixed.ld
+  link.ld
+  stdcart.ld
+TYPE LIB)
+
+# do not create crt0, that will precedence over the common version
+# which we depending on; rather create something specific
+add_platform_library(atari8-stdcart-c
```

Comment:
This does need to be in crt0, since it's necessary to enter even a freestanding C environment. The contents of the crt0.o defined in the parent atari8 target should automatically be merged into a platform library of the same name defined here in the child.